File: test/test/ViewController.m
//
//  ViewController.m
//  test
//
//  Created by Mia on 2018/2/9.
//  Copyright © 2018年 Mia. All rights reserved.
//

#import "ViewController.h"
#import "A_VC.h"
#import "B_VC.h"


@interface ViewController ()

@end

@implementation ViewController

- (void)viewDidLoad {
    [super viewDidLoad];
    A_VC *A = [A_VC new];
    B_VC *B = [B_VC new];
    [self addChildViewController:A];
    [self addChildViewController:B];
    
    A.view.frame = CGRectMake(0, 0, 300, 200);
    B.view.frame = CGRectMake(0, 200, 300, 200);
    [self.view addSubview:A.view];
    [self.view addSubview:B.view];
}


- (void)didReceiveMemoryWarning {
    [super didReceiveMemoryWarning];
    // Dispose of any resources that can be recreated.
}


@end

File: ChardVideo/ChardVideo/ViewController.swift
//
//  ViewController.swift
//  ChardVideo
//
//  Created by ChardLl on 2017/10/27.
//  Copyright © 2017年 com.chard. All rights reserved.
//

import UIKit

class ViewController: UIViewController {

    override func viewDidLoad() {
        super.viewDidLoad()
        // Do any additional setup after loading the view, typically from a nib.
//        let playerHeight = view.bounds.size.width * 9 / 16
//        let playerView = VideoPlayerView(frame: CGRect(x: 0, y: 20, width: view.bounds.size.width, height: playerHeight))
//        view.addSubview(playerView)
    }
    
    let videoTool: videoLauncher = videoLauncher()
    
    @IBAction func launchVideo(_ sender: UIButton) {
        print("...")
        
        videoTool.showVideo { (isComplete) in
            print("complete")
        }
    }
}



